editor/import/resource_importer_scene.cpp

```diff
@@ -2528,6 +2529,11 @@ Error ResourceImporterScene::import(const String &p_source_file, const String &p
 
 	progress.step(TTR("Saving..."), 104);
 
+	int flg = 0;
+	if (EDITOR_GET("filesystem/on_save/compress_binary_resources")) {
+		flg |= ResourceSaver::FLAG_COMPRESS;
+	}
```

Comment:
Is `flags` already taken? I think we have space to use a fully spelled out variable name :slightly_smiling_face:

Reply:
oop, sure thing. Let me give that a cursory compile and I'll push it up right now.